central/telemetry/service/service_impl.go

```diff
@@ -24,9 +28,26 @@ var (
 		user.With(permissions.Modify(resources.DebugLogs)): {
 			"/v1.TelemetryService/ConfigureTelemetry",
 		},
+		anyAuthenticated{}: {
```

Comment:
I guess `user.With()` is what you want here. Filed https://github.com/stackrox/stackrox/pull/4209.